src/bin/pg_autoctl/cli_do_monitor.c

```diff
@@ -561,3 +571,30 @@ cli_monitor_resume_node(int argc, char **argv)
 	log_info("Node %s:%d will exit from maintenance state soon",
 			 keeper.config.nodename, keeper.config.pgSetup.pgport);
 }
+
+
+/*
+ * cli_monitor_version checks and displays the monitor extension version.
```

Comment:
This comment is not reflecting what we are doing.

the call to `monitor_ensure_extension_version` has side effects that it tries to upgrade as well. I think it is undesirable to have a command that should print the installed version to try and change the version installed.

Could we make this command adhere to the comment on this function? eg. show the version, without ensuring it is the version we are expecting given the version of `pg_autoctl`.

EDIT: I now see this is under the do commands, in that case we can decide that it will update to the right version, but that should be reflected both in this comment, and in the `--help` of the cli command

Reply:
Good point, fixed the comments and the help message that way:

```
$ PG_AUTOCTL_DEBUG=1 pg_autoctl do monitor 
Available commands:
  pg_autoctl do monitor
  + get       Get information from the monitor
    register  Register the current node with the monitor
    active    Call in the pg_auto_failover Node Active protocol
    pause     Pause pg_auto_failover on this node
    resume    Resume pg_auto_failover on this node
    version   Check that monitor version is 1.0; alter extension update if not
```